### ISSUE TITLE
Font license 

### ISSUE THREAD

**Issue by srmyers17** [department-of-veterans-affairs/va.gov-team#13806]
# User Story

As a designer/developer for the VA mobile app, I want to know the LOE for procuring a license to use VA-specific fonts in the app so that I can decide whether this is an MVP priority.

**Comment by bridgethapner** [comment]
Both Bitter & Source are licensed under this license (https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL). I looked for clarification about native mobile licensing in here and didn't find anything. The licenses displayed on Google Fonts for [Bitter](https://fonts.google.com/specimen/Bitter) and [Source Sans](https://fonts.google.com/specimen/Source+Sans+Pro) read: "These fonts are licensed under the Open Font License. You can use them freely in your products & projects - print or digital, commercial or otherwise. However, you can't sell the fonts on their own. This isn't legal advice, please consider consulting a lawyer and see the full license for all details."

----

**TLDR: It's probably fine to use these fonts. IMO if we go down this route we should talk to a lawyer like Google suggests.**

**Comment by bridgethapner** [comment]
**Capturing notes from talking about font stuff:** 

## Where we're at
The big question here is **licensure** - we think rules regarding web and native apps are different rules & want to be thorough and understand what licensing we need to comply with legal requirements.

### Next steps: 
- @artsymartha68 reaching out to other mobile app teams at VA to see how they handle font licensing. 
- @rtwell has reached out to Patrick Bateman to ask about legal stuff in general. 

## Assumptions:
- We'll have to put attribution to fonts in the app somewhere for November. 
- Since we're launching internally only in November, it may be okay to hammer out any open licensing questions for app store release not for November.

**Comment by artsymartha68** [comment]
Update: We have the OK from Patrick per @rtwell that we are OK to distribute the fonts.

Steve K said we can consult with our DP experts as to where within the app, on the store page, etc. these licenses need to appear.

**Comment by bridgethapner** [comment]
Thanks for confirming @artsymartha68! Closing this ticket, and will open a new one for determining where to display licenses in the app.

**Comment by srmyers17** [comment]
@bessgreen FYI
